src/providers/sh/commands/deploy/deploy.js

```diff
@@ -149,6 +151,8 @@ const help = () => {
     'FILE'
   )}         Include env vars from .env file. Defaults to '.env'
     -C, --no-clipboard             Do not attempt to copy URL to clipboard
+    --json                         Output deployment info as JSON to stdout
+    --raw                          Output the deployment URL only to stdout
```

Comment:
Curious why `raw` is the name of the flag that outputs the deployment URL to stdout. Is that a convention used elsewhere? My instinct would be to name this something more obvious like `output-deployment-url`.

Reply:
Given that they're mutually exclusive, `--format` does indeed make more sense to me.